Dockerfile

```
@@ -28,13 +28,34 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN pip install --no-cache-dir \
+    "numpy==1.23.5" \
+    "pandas>=2.0.0,<2.1.0" \
+    "scipy>=1.10.0,<1.11.0" \
+    "scikit-learn>=1.2.0,<1.4.0" \
+    "h5py>=3.8.0,<3.10.0" \
+    "seaborn>=0.12.0" \
+    "scikit-image>=0.20.0"
+
+# Install other py-feat dependencies
+# Explicitly add xgboost
+RUN pip install --no-cache-dir \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)